third_party/ots/include/ots-memory-stream.h

```diff
@@ -24,14 +24,7 @@ class MemoryStream : public OTSStream {
       return false;
     }
     std::memcpy(static_cast<char*>(ptr_) + off_, data, length);
-#if defined(_MSC_VER)
-#pragma warning(push)
-#pragma warning(disable : 4267)  // possible loss of data
-#endif
-    off_ += length;
-#if defined(_MSC_VER)
-#pragma warning(pop)
-#endif
+    off_ += static_cast<off_t>(length);
```

Comment:
Since this is a modification relative to upstream
can we mark it with an EOL comment like
`    off_ += static_cast<off_t>(length);  // Cobalt`
optionally with a colon and brief explanation?
Comments on diffs help during updates.

[same on line 81 below]

Reply:
This isn't something that we have been doing up until now, and when i look at the potential issues that it could bring i'm not sure it's a pattern we could consistently apply outside of trivial changes. We should have a discussion about this outside of this PR.

Reply:
Most of our diffs are marked with preprocessor conditionals
that serve the same (and sometimes no other) purpose.
We do also have comments on diffs in some places and
I'm definitely curious what issues those might cause
when we come to that discussion.

Reply:
Resolving as is for now to get this fixed, let's follow up separately.